models/comments-by-id.model.js

```diff
@@ -12,6 +13,20 @@ exports.selectCommentsById = (article_id) => {
             });
           }
         return result.rows;
-    })
-}
+    });
+};
+
+exports.insertCommentById = ( { article_id, author, body }) => {
+  if(!article_id || !author || !body) {
```

Comment:
I see what you are doing here but if there is no article id you won't get to this model as it would be different endpoint and picked up by your generic 404 error handler

Reply:
Got it yes